proposals/NNNN-Add-RPC-Conflict-Management.md

```diff
@@ -0,0 +1,218 @@
+# Add RPC Conflict Management
+
+* Proposal: [SDL-NNNN](NNNN-Clarification-of-SDL-App-Icon-display-sequence-and-display-order.md)
+* Author: [Akihiro Miyazaki (Nexty)](https://github.com/Akihiro-Miyazaki)
+* Status: **Awaiting review**
+* Impacted Platforms: [ Core / SDL Documents ]
```

Comment:
Please remove `SDL Documents` as this is not a possible platform per the proposal template: https://github.com/smartdevicelink/sdl_evolution/blob/master/0000-template.md.  If there are any other platforms in the template that should be included here, please add.

Reply:
Hi. @theresalech -san,
Thank you for your review.
I will remove the word "SDL Documents" and make a commit it.
Could you please confirm it again?
Thank you.